setup.py

```diff
@@ -72,6 +72,16 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
+
+extra_require = {
+    'gevent':  ["gevent>=0.13"],
```

Comment:
Style nit: We usually try to stick with the most used quoting style in a file. For `setup.py`, it looks like we went with single quotes. It would be nice use it in new code.